audio_video_recorder_server/RecordTask task
---
bool success
string message
